File: БР1.2/Соболев Артём/labs/lab1/FitnessAPI/src/main/java/io/github/artsobol/fitnessapi/feature/auth/auth/dto/request/LoginRequest.java
package io.github.artsobol.fitnessapi.feature.auth.auth.dto.request;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Size;

public record LoginRequest(
        @Size(max = 64, message = "auth.user.username.long")
        @NotBlank(message = "auth.user.username.blank")
        String username,

        @NotBlank(message = "auth.user.password.blank")
        String password
) {
}

File: БР1.2/Алексеев Андрей/labs/lab1/src/main/kotlin/ru/itmo/pxdkxvan/lab1/auth/dto/LoginRequest.kt
package ru.itmo.pxdkxvan.lab1.auth.dto

import jakarta.validation.constraints.Email
import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.Size

data class LoginRequest(
    @field:Email
    @field:NotBlank
    @field:Size(max = 255)
    val email: String,
    @field:NotBlank
    @field:Size(min = 8, max = 255)
    val password: String,
)
